### ISSUE TITLE
Vault Exit Portals Invisible

### ISSUE THREAD

**Issue by OrdoFlammae** [simon987/Much-Assembly-Required#220]
The Vault Exit Portals look no different than any normal square in the Vault. In other words, they're invisible. However, they're still functional in every other way.

**Comment by simon987** [comment]
Are they 100% invisible in the client (even when you hover them) ?

**Comment by OrdoFlammae** [comment]
Yup. I figured out why, though. They have no representation in the typescript (see GameObject.js:60-87). If you check for a VaultExitPortal class type, and just create a Portal, then it works fine. Some work should be spent on this, but it's a temporary fix.
